src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Log/FunctionId.cs

```diff
@@ -484,6 +484,8 @@ internal enum FunctionId
     Renamer_FindRenameLocationsAsync = 387,
     Renamer_ResolveConflictsAsync = 388,
 
+    Workspace_EventsImmediate = 389,
```

Comment:
I would keep this range free for rename

Reply:
Didn't see a good spot for a one-off id to add for Workspace events. Any suggestion?

Reply:
I'd just start a new range :-)